build.gradle.kts

```diff
@@ -144,9 +144,6 @@ subprojects {
     register<Test>("testFireblocksE2E") {
```

Comment:
Do we still need this anymore?

Reply:
I am not sure but I don't think it is being used. I will clean up.